src_feedback/elf.rs

```diff
@@ -0,0 +1,666 @@
+use std::mem;
+use std::fs::{File};
+use std::io::{Error, ErrorKind, Read, Seek, SeekFrom};
+use std::borrow::BorrowMut;
+
+use std::cell::RefCell;
+
+use crate::tools::{search_address_opt_key, extract_string};
+
+const EI_NIDENT: usize = 16;
```

Comment:
Does it make sense to use the definitions from the [rust elf crate](https://crates.io/crates/elf)? It looks like they define [EI_IDENT](https://cole14.github.io/rust-elf/elf/types/constant.EI_NIDENT.html), [ET_NONE](https://cole14.github.io/rust-elf/src/elf/types.rs.html#165), etc.

Reply:
I am considering using existing ELF parsers, for example, elf crate.  But, I still need to do some studies about performance.
These definitions are copied from elf.h with a small script.